lang/main.json

```diff
@@ -355,7 +355,8 @@
         "toggleScreensharing": "Switch between camera and screen sharing",
         "toggleShortcuts": "Show or hide keyboard shortcuts",
         "videoMute": "Start or stop your camera",
-        "videoQuality": "Manage call quality"
+        "videoQuality": "Manage call quality",
+        "toggleTileView": "Toggle tile view"
```

Comment:
Sorry, but I just noticed that we already have this string as two more keys in the translation. You can use here `toolbar.tileViewToggle`, this way there will be no need for translation of the already translated string. Thanks.

Reply:
Done, used translations of toolbar.tileViewToggle.